mantis-server/mantis-server-agent/build.gradle

```diff
@@ -15,8 +15,21 @@
  */
 
 dependencies {
-    compile project(":mantis-control-plane:mantis-control-plane-core")
+    implementation project(":mantis-runtime-loader")
+    implementation project(":mantis-server:mantis-server-worker-client")
```

Comment:
Isn't this a transitive dependency? Do we need to define this explicitly?

Reply:
this is "server-worker-client" and the loader only depends on "control-plane-client".